libs/blocks/aside/aside.css

```diff
@@ -370,19 +448,84 @@
   justify-content: flex-start;
 }
 
+.aside.split .foreground.container .text .icon-stack-area {
+  display: flex;
+  flex-flow: row wrap;
+  flex-direction: column;
+  gap: 12px;
+  margin: -8px 0 var(--spacing-s);
+  width: 100%;
+  padding: 0;
+  list-style-type: none;
+}
+
+.aside.split .foreground.container .text .icon-stack-area li,
+.aside.split .foreground.container .text .icon-stack-area li a {
+  display: flex;
+  align-items: center;
+  gap: 12px;
+  font-weight: bold;
+}
+
+.aside.split .foreground.container .text .icon-stack-area li picture {
+  flex-shrink: 0;
+}
+
+.aside.center:not(.notification):not(.split) .foreground.container {
+  padding: 0;
+}
+
+.aside.center:not(.notification):not(.split) .foreground.container .text {
+  margin: 80px 0;
+  text-align: center;
+  padding: 0;
+}
+
+.aside.center:not(.notification):not(.split) .foreground.container .icon-area {
+  max-width: 100%;
+}
+
+.aside.center:not(.notification):not(.split) .foreground.container .text .action-area {
+  justify-content: center;
+}
+
+.aside:not(.notification) .foreground.container .text .detail-m {
+  margin-bottom: var(--spacing-xs);
+}
+
+.aside.split .image.format {
+  display: flex;
+}
+
+.aside.split.bio .foreground.container .text .icon-area {
+  display: none;
+}
+
+
+.aside.center:not(.notification):not(.split) .foreground.container .text .icon-area {
+  height: var(--icon-size-xxl);
+}
+
+.aside.center:not(.notification):not(.split) .foreground.container .text .icon-area img {
+  height: var(--icon-size-xxl);
+  max-width: 300px;
+  width: auto;
+}
+
```

Comment:
quite a few `:not(.notification)` and `:not(.notifcation):not(.split)` in lines 474 - 514. Is it possible to move some of these selectors up and `.notification` below, so `.notification` can be additive?

Reply:
@JasonHowellSlavin I have made the changes to remove the nots where ever possible. The not for notification are used because there are two center variations (aside (new request) and aside notification (existing block live on production milo pages) ). The specs for both are different and if we do not differentiate the center then certain style set of aside center gets applied to notification center..